tests/conftest.py

```diff
@@ -478,3 +479,27 @@ def cluster_sanity_scope_session(
         dsci_resource=dsci_resource,
         junitxml_property=junitxml_plugin,
     )
+
+
+@pytest.fixture(scope="session")
+def fail_if_missing_dependent_operators(admin_client: DynamicClient) -> None:
+    missing_operators: list[str] = []
+
+    for operator_name in py_config.get("dependent_operators", []).split(","):
+        LOGGER.info(f"Verifying if {operator_name} is installed")
+        for csv in ClusterServiceVersion.get(
```

Comment:
you are making the same api call in the for loop. You might want to save the CSVs outside the for loop and then traverse through the list inside.

Reply:
done